Framework/API/inc/MantidAPI/MatrixWorkspace.h

```diff
@@ -315,10 +315,10 @@ class MANTID_API_DLL MatrixWorkspace : public IMDWorkspace,
 
   /**
    * Probes if DX (X Error) values were set on a particular spectrum
-   * @param index: the spectrum index
+   * @param specNum: the spectrum number
    */
-  virtual bool hasDx(const std::size_t index) const {
-    return getSpectrum(index)->hasDx();
+  virtual bool hasDx(const std::size_t specNum) const {
```

Comment:
I think this should still say `index`. It is the 0-based value for the vector lookup.